Add template for pull request (#5)
kristinejosami/first-git-project: .github/pull_request_template.md

# Description

# Checklist:

> - [ ] My code follows the style guidelines of this project(Example) 

Reviewer: ```suggestion
- [ ] My code follows the style guidelines of this project example
```

Reviewer: @naokitcut please check 😸

> - [ ] My code follows the style guidelines of this project(Expample) 

Reviewer: ```suggestion
- print("My code follows the style guidelines of this project(Example)")
```

Author: @kristinejosami  Oh, sorry, I'm using markdown...
Can you check it and give me another suggestion?

https://guides.github.com/features/mastering-markdown/

> - [ ] My code follows the style guidelines of this project(Expample) 

Reviewer: ```suggestion
- [ ] My code follows the style guidelines of this project(Example) 
```
